src/main/java/org/cru/redegg/reporting/InMemoryErrorQueue.java

```diff
@@ -82,6 +93,22 @@ public void shutdown()
     @Override
     public void enqueue(final ErrorReport report)
     {
+        try
+        {
+            submit(report);
+        }
+        catch (RejectedExecutionException e)
+        {
+            System.out.println("submission rejected");
```

Comment:
Do you log with `System.out.println`, as opposed to some `logger`, so as not to include that log in the error report?